src/lib/Containers/Inquiry.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { MetadataText, SmallHeadline } from "../Components/Inbox/Typography"
+
+import {
+  FlatList,
+  ImageURISource,
+  KeyboardAvoidingView,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  ViewProperties,
+} from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../data/colors"
+import fonts from "../../data/fonts"
+
+import BottomAlignedButton from "../Components/Consignments/Components/BottomAlignedButton"
+
+import ArtworkPreview from "../Components/Inbox/Conversations/ArtworkPreview"
+import ARSwitchBoard from "../NativeModules/SwitchBoard"
+import { gravityURL } from "../relay/config"
+
+const Container = styled.View`
+  flex: 1
+  flex-direction: column
+`
+const Header = styled.View`
+  alignSelf: stretch
+  margin-top: 10
+  flex-direction: column
+  margin-bottom: 30
+`
+// This is really rubbish, but I basically have to create an equally sized element
+// on the top right, to get the title in the middle
+const PlaceholderView = styled(SmallHeadline)`
+  padding-right: 20
+  color: white
+`
+const TitleView = styled.View`
+  align-self: center
+  align-items: center
+  margin-top: 6
+`
+const HeaderTextContainer = styled.View`
+  flex-direction: row
+  justify-content: space-between
+`
+const CancelButton = styled.TouchableOpacity`
+  padding-left: 20
+`
+const Content = styled.View`
+  margin-left: 20
+  margin-right: 20
+`
+const InquiryTextInput = styled.TextInput`
+  font-size: 16
+  margin-top:20
+  font-family: ${fonts["garamond-regular"]}
+`
+const ResponseRate = styled(SmallHeadline)`
+  color: ${colors["yellow-bold"]}
+  marginTop: 5
+`
+const ResponseIndicator = styled.View`
+  width: 8
+  height: 8
+  border-radius: 4
+  margin-top: 5
+  margin-right: 5
+  background-color: ${colors["yellow-bold"]}
+`
+const ResponseRateLine = styled.View`
+  flex: 1
+  flex-direction: row
+  align-items: center
+  min-height: 12
+  margin-top: 5
+`
+
+export class Inquiry extends React.Component<RelayProps, any> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      text: this.props.inquiry.partner.contact_message,
+    }
+  }
+
+  dismissModal() {
+    ARSwitchBoard.dismissModalViewController(this)
```

Comment:
@orta We moved away from (ES7) static class properties at some point because Chrome wasn’t able to properly debug them and so we deemed them too experimental for now.